heat/preprocessing/tests/test_preprocessing.py

```diff
@@ -172,75 +238,110 @@ def test_MaxAbsScaler(self):
+                                quantile_range=(75.0, 23.0),
+                                copy=copy,
+                                with_centering=with_centering,
+                                with_scaling=with_scaling,
+                            )
+                        Z = ht.zeros(
+                            (MPI.COMM_WORLD.Get_size() * 10, MPI.COMM_WORLD.Get_size() * 2),
+                            split=split,
+                        )
+                        with self.assertRaises(ValueError):
+                            scaler.transform(Z)
+                        # second case: use in-place operations
+                        # PROBLEM: THIS DOES NOT WORK AS EXPECTED...
+                        copy = False
+                        X = _generate_test_data_set(
+                            MPI.COMM_WORLD.Get_size() * 10,
+                            MPI.COMM_WORLD.Get_size() * 4,
+                            split=split,
+                            dtype=ht.float32,
+                        )
+                        X_cpy = X.copy()  # for comparison only
+                        scaler = ht.preprocessing.RobustScaler(
+                            quantile_range=(24.0, 76.0),
+                            copy=copy,
+                            with_centering=with_centering,
+                            with_scaling=with_scaling,
+                        )
+                        scaler.fit(X)
+                        X = scaler.transform(X)
+                        if with_centering:
+                            self.assertTrue(
+                                ht.allclose(
+                                    ht.median(X, axis=0), ht.zeros(X.shape[1]), atol=atol_fit
                                 )
-                                scaler = ht.preprocessing.RobustScaler(
-                                    quantile_range=(75.0, 23.0),
-                                    copy=copy,
-                                    with_centering=with_centering,
-                                    with_scaling=with_scaling,
+                            )
+                        if with_scaling:
+                            self.assertTrue(
+                                ht.allclose(
+                                    ht.percentile(X, 76.0, axis=0)[:-2]
+                                    - ht.percentile(X, 24.0, axis=0)[:-2],
+                                    ht.ones(X.shape[1])[:-2],
+                                    atol=atol_fit,
                                 )
-                            Z = ht.zeros(
-                                (MPI.COMM_WORLD.Get_size() * 10, MPI.COMM_WORLD.Get_size() * 2),
-                                split=split,
                             )
-                            with self.assertRaises(ValueError):
-                                scaler.transform(Z)
+                        Y = scaler.inverse_transform(X)

```

Comment:
Here, we also should find out why the robust scaler could not be realized in the same way as the other scalers for the in-place variant.